experimental/adaptive-dialog/csharp_dotnetcore/todo-bot/Dialogs/ViewToDoDialog/ViewToDoDialog.lg

```diff
@@ -2,25 +2,35 @@
 > See https://github.com/Microsoft/BotBuilder-Samples/tree/master/experimental/language-generation to learn more
 
 > You can now add reference to other .lg files.
-[Root LG](..\RootDialog\RootDialog.lg)
+[Root LG](../RootDialog/RootDialog.lg)
```

Comment:
Is the back\ forward slash a Windows / Mac thing? Was this intentional?

Reply:
The prior iteration was not cross-platform compatible. The new notation is - works on both mac and windows.